caddytls/setup.go

```diff
@@ -164,6 +164,22 @@ func setupTLS(c *caddy.Controller) error {
 					return c.Errf("Unsupported Storage provider '%s'", args[0])
 				}
 				config.StorageProvider = args[0]
+
+			case "http2":
+				args := c.RemainingArgs()
+				if len(args) != 1 {
+					return c.ArgErr()
+				}
+
+				switch args[0] {
+				case "off":
+					config.DisableHTTP2 = true
+				case "on":
+					config.DisableHTTP2 = false
```

Comment:
Is there a case where HTTP/2 would need to be turned on, since it's enabled by default? (I would rather not have people thinking they need to explicitly turn on HTTP/2.) If not, let's remove the "on" option.